src/comp.ts

```diff
@@ -71,7 +71,7 @@ function toChecksumAddress(_address) {
  */
 export async function getCompBalance(
   _address: string,
-  _provider : Provider | string='mainnet'
+  _provider : Provider | string='rsk-testnet'
```

Comment:
```suggestion
  _provider : Provider | string='testnet'
```
Use same name as in constants